lib/src/io.dart

```diff
@@ -390,18 +390,20 @@ void _attempt(String description, void Function() operation,
     return null;
   }
 
-  for (var i = 0; i < 3; i++) {
+  const maxRetries = 10;
+  for (var i = 0; i < maxRetries; i++) {
     try {
       operation();
       break;
     } on FileSystemException catch (error) {
       var reason = getErrorReason(error);
       if (reason == null) rethrow;
 
-      if (i < 2) {
+      if (i < maxRetries - 1) {
+        final delayMilliseconds = 50 * (i + 1);
         log.io('Pub failed to $description because $reason. '
-            'Retrying in 50ms.');
-        sleep(Duration(milliseconds: 50));
```

Comment:
this is low cost, we can just do it every 50ms and give it more retries...

Reply:
Done